galaxy_api/api/models.py

```diff
@@ -27,7 +27,7 @@ class Namespace(models.Model):
 
     # Fields
 
-    name = models.CharField(max_length=64, unique=True, editable=False)
+    name = models.CharField(max_length=64, unique=True, editable=True)
```

Comment:
It may require updating migrations as well.
Also `editable` is `True` by default, so please remove it.

Reply:
Because this change does not affect SQL schema, it's acceptable \ preferrable to fix already existing migration.